proxy/upgrade_test.go

```diff
@@ -359,8 +360,9 @@ func TestAuditLogging(t *testing.T) {
 			defer dc.Close()
 
 			rt := routing.New(routing.Options{
-				DataClients: []routing.DataClient{dc},
-				Log:         tl,
+				DataClients:    []routing.DataClient{dc},
+				PostProcessors: []routing.PostProcessor{routing.NewEndpointsRegistry(endpoints.NewRegistry(endpoints.RegistryOptions{}))},
```

Comment:
Why do we need registry in this test?

Reply:
Was added while experimenting with registry. Of course, it's not needed here, removed.